pkgs/top-level/python-packages.nix

```diff
@@ -7686,10 +7692,16 @@ in {
 
   spacy = callPackage ../development/python-modules/spacy { };
 
+  spacy-alignments = callPackage ../development/python-modules/spacy-alignments { };
+
   spacy-legacy = callPackage ../development/python-modules/spacy/legacy.nix { };
 
   spacy_models = callPackage ../development/python-modules/spacy/models.nix { };
 
+  spacy_pkuseg = callPackage ../development/python-modules/spacy_pkuseg { };
```

Comment:
Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)

Reply:
Thanks! Fixed. I also changed the license to `mit`. In contrast to upstream `pkuseg`, `spacy-pkuseg` has redistributable data:

https://github.com/explosion/spaCy/issues/6214#issuecomment-704739719